tests/wicked/wlan/sut/t01_open_static_ip.pm

```diff
@@ -0,0 +1,67 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test WiFi setup with wicked (Open static IP)
+# - WiFi Access point:
+#   - Use virtual wlan devices
+#   - AP with hostapd is running in network namespace
+# - WiFi Station:
+#   - connect using ifcfg-wlan1 and `wicked ifup`
+#   - check if STA is associated to AP
+#   - ping both directions AP <-> STA
+#
+# Maintainer: cfamullaconrad@suse.com
+
+
+use Mojo::Base 'wicked::wlan';
+use testapi;
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+
+    # Setup ref
+    $self->netns_exec('ip addr add dev wlan0 ' . $self->ref_ip . '/24');
+    $self->spurt_file('hostapd.conf', <<EOTEXT);
+ctrl_interface=/var/run/hostapd
```

Comment:
while generally I like idea of dropping use of data folder and keep config files in test itself . I think this is too much , can we move actual content out of run function at least ? it is hard to understand what actually test is doing and where test code and where is file content

Reply:
Yeah. I will make two functions for creating the `hostapd.conf` and `wpa_supplicant.conf` content. 
But I would like to keep it in the test-module and not in the `wicked::wlan` cause I liked also the idea to see what the config file looks like, by just looking at the testcode.

The other idea is, we would upload each configfile.

Reply:
actually I would be also against to move it out from this file. In this case I would prefer files in data folder and upload them . I just want get them out from run function so I can read test flow without greping config files . So dedicated functions in same file will be perfect